<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cave</class>
 <widget class="QWidget" name="Cave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cave</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>514</y>
     <width>487</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="gen_mode_text">
        <property name="text">
         <string>Режим отрисовки генераций:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="gen_mode_manual">
        <property name="text">
         <string>Пошаговый</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="gen_mode_auto">
        <property name="text">
         <string>Автоматический</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="mil_step_text">
        <property name="text">
         <string>Сколько милисекунд между шагами:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="mil_step">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_step">
        <property name="text">
         <string>Следующий шаг</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="limit_text">
          <property name="text">
           <string>Предел «рождения»:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSpinBox" name="limit_birth">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="limit_death_text">
        <property name="text">
         <string>«смерти»:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="limit_death">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="init_chance_text">
          <property name="text">
           <string>Шанс инициализации:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="init_chance">
          <property name="value">
           <number>35</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="create_cave">
        <property name="text">
         <string>Создать пещеру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="size_cave_text">
        <property name="text">
         <string>Размер пещеры:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="size_cave_x">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="size_cave_y">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="file">
        <property name="text">
         <string>Из файла</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
